### PR TITLE
SAR-3700: Create agent Capture Nino controller and view

### PR DESCRIPTION
**Pull request by LZaharia**

### CODE REVIEW

File: it/uk/gov/hmrc/vatsignupfrontend/controllers/agent/soletrader/CaptureNinoControllerISpec.scala [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    super.beforeEach()
+    enable(SkipCidCheck)
+  }
+
+  override def afterEach(): Unit = {
+    super.afterEach()
+    disable(SkipCidCheck)
+  }
+
+  val url = "/national-insurance-number"
```
Comment (r-melvin) [inline it/uk/gov/hmrc/vatsignupfrontend/controllers/agent/soletrader/CaptureNinoControllerISpec.scala]: this val isn't used in the test

File: test/uk/gov/hmrc/vatsignupfrontend/assets/MessageLookup.scala [hunk 2]
```diff
@@ -514,7 +514,11 @@ object MessageLookup {
     val title = heading + ServiceName.principalSuffix
     val formHint = "It's on your National Insurance card, benefit letter, payslip or P60. For example 'QQ 12 34 56 C'."
   }
-
+  object AgentCaptureNino {
+    val heading: String = "What is your client's National Insurance number?"
+    val title = heading + ServiceName.agentSuffix
+    val formHint = "It's on your National Insurance card, benefit letter, payslip or P60. For example, 'QQ 12 34 56 C'."
```
Comment (r-melvin) [inline test/uk/gov/hmrc/vatsignupfrontend/assets/MessageLookup.scala]: need to query this content, doesn't seem right for agents

File: it/uk/gov/hmrc/vatsignupfrontend/controllers/agent/soletrader/CaptureNinoControllerISpec.scala (hunk 1 above)
Comment (LZaharia) [reply it/uk/gov/hmrc/vatsignupfrontend/controllers/agent/soletrader/CaptureNinoControllerISpec.scala]: The val is now used

File: test/uk/gov/hmrc/vatsignupfrontend/assets/MessageLookup.scala (hunk 2 above)
Comment (LZaharia) [reply test/uk/gov/hmrc/vatsignupfrontend/assets/MessageLookup.scala]: I'm waiting for Ali or Parool to get back on the JIRA ticket

**Pull request merged**